beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/SyncSourceFactory.java

```diff
@@ -36,11 +40,20 @@ public SyncSourceFactory(final AsyncRunner asyncRunner, final TimeProvider timeP
     this.timeProvider = timeProvider;
   }
 
-  public SyncSource getOrCreateSyncSource(final Eth2Peer peer) {
+  public SyncSource getOrCreateSyncSource(final Eth2Peer peer, final Spec spec) {
     // Limit request rate to just a little under what we'd accept
-    final int maxBlocksPerMinute = MAX_BLOCKS_PER_MINUTE - SYNC_BATCH_SIZE.intValue() - 1;
+    final int maxBlocksPerMinute = MAX_REQUEST_BLOCKS - SYNC_BATCH_SIZE.intValue() - 1;
```

Comment:
It's not clear why we use `MAX_REQUEST_BLOCKS` as a `maxBlockPerMinute`? May be it makes sense to change `MAX_BLOCKS_PER_MINUTE` constant?

Reply:
@Nashatyrev @mehdi-aouadi ok, I think we should discuss how to deal with constants like this with the team. @StefanBratanov don't like some current constants things also